admiral/pkg/clusters/serviceentry.go

```diff
@@ -2142,7 +2166,13 @@ func createAdditionalEndpoints(
 	}
 	virtualService.Annotations = vsAnnotations
 
-	err = addUpdateVirtualService(ctxLogger, ctx, virtualService, existingVS, namespace, rc, rr)
+	if len(virtualService.Spec.Hosts) == 0 {
```

Comment:
Can we remove this check as well, as hosts are verified at https://github.com/istio-ecosystem/admiral/pull/390/files#diff-fcb33638fb967d38496fdd069b6db31d4d879df2a93e0d47791cc97cdc55948bR2097